src/instructlab/model/evaluate.py

```diff
@@ -622,17 +636,34 @@ def evaluate(
                     server.shutdown()
 
             max_score = get_benchmark_max_score(Benchmark.MT_BENCH)
-            print("# SKILL EVALUATION REPORT")
-            print("\n## MODEL (SCORE)")
-            display_model(model, overall_score, max_score)
-            print(f"\n### TURN ONE (0.0 to {max_score}):")
-            print(round(turn_scores[0], 2))
-            print(f"\n### TURN TWO (0.0 to {max_score}):")
+            turn1_score = turn_scores[0]
             turn2_score = turn_scores[1]
-            if isinstance(turn2_score, float):
-                turn2_score = round(turn2_score, 2)
-            print(turn2_score)
-            display_error_rate(error_rate)
+            turn1_score_rounded = round(turn1_score, 2)
+            turn2_score_rounded = round(turn2_score, 2)
+            if format == "json":
+                json_data = {
+                    "model": model,
+                    "overall_score": overall_score,
+                    "max_score": max_score,
+                    "turn1_score": turn1_score,
+                    "turn1_score_rounded": turn1_score_rounded,
+                    "turn2_score": turn2_score,
+                    "turn2_score_rounded": turn2_score_rounded,
+                    "error_rate": error_rate,
+                }
+                print(json_data)
```

Comment:
We're going to want to use `json.dumps()` here to ensure what we're outputting here is standards compliant.